fedWireMessage.go

```diff
@@ -131,6 +137,8 @@ type FEDWireMessage struct {
 	RemittanceFreeText *RemittanceFreeText `json:"remittanceFreeText,omitempty"`
 	// ServiceMessage
 	ServiceMessage *ServiceMessage `json:"serviceMessage,omitempty"`
+
+	validateOpts *ValidateOpts
```

Comment:
I think it might make more sense for this field to be one level up, on `File`. If it's here as an unexported field, is there any way for someone to specify validation options in a JSON request?

Reply:
I was trying to keep records only into json file, don't to expose the validation option
To use validation option is for package side after json Marshal

Reply:
In ACH we pass the ValidateOpts down to nested structs where needed. Users of the Go library and HTTP server need to be able to set ValidateOpts.

Reply:
that' right, changed validateOpts  as public field